docs/modules/ROOT/pages/reference/concepts.adoc

```diff
@@ -11,6 +11,12 @@ low to high priority):
 
 * Component parameter default values
 * Global generic default values
+
+That global generic default allows then to include further classes and build up any hierarchy.
```

Comment:
With the change the whole structure of this section is pretty confusing, as the configuration value sources can now arbitrarily be changed.

I think the part of the section which starts with "Commodore defines the following hierarchy" needs to be completely replaced with a description of the new dynamic inventory mechanism, and a description of how component defaults fit into the new picture.

Reply:
I have rewritten that part.